src/composables/staking/useStaking.ts

```diff
@@ -0,0 +1,25 @@
+import { DecoratedPoolWithShares } from '@/services/balancer/subgraph/types';
+
+export enum StakeState {
+  CanStake = 'can_stake',
+  MaxStaked = 'max_staked',
+  NoGuage = 'no_guage'
```

Comment:
Minor - Seems like we got a couple of ways to define enums, personally I like to see them as:
```
export enum StakeState {
  CAN_STAKE = 'canStake,
  MAX_STAKED = 'maxStaked',
  NO_GAUGE = 'noGuage'
```

Reply:
oki

Reply:
opting to go with capital case